sycl/source/detail/context_impl.hpp

```diff
@@ -98,6 +98,20 @@ class context_impl {
   /// \return true if this context is a host context.
   bool is_host() const;
 
+  /// Returns the backend associated with this context.
+  ///
+  /// \return the backend associated with this context.
+  backend get_backend() const noexcept {
+    backend Result;
+    if (is_host())
+      Result = backend::host;
+    else {
```

Comment:
No need for curly braces for one-liner.
See https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
Corrected